sympy/printing/defaults.py

```diff
@@ -1,20 +1,7 @@
 from __future__ import print_function, division
 
-class DefaultPrinting(object):
```

Comment:
Oh I never realized this was here. Why not just put the new class in this file, instead of creating a new file?

Reply:
Or better just add the methods to this class. No need to rename things.

Reply:
We cant use this class in core/basic, as it creates circular imports - so we have no choice but to move it to a file somewhere in core..

I figured since no one really knew that this existed, it was a good opportunity to pick a more understandable name. I'm can change it back if you want me to, but I'd argue the new name is way clearer. Using gerunds as a class name is very odd.